gpii/node_modules/lifecycleManager/src/LifecycleManagerSession.js

```diff
@@ -65,6 +71,15 @@ var gpii = fluid.registerNamespace("gpii");
         }
     });
 
+    // Only save prefs to the cloud when the preferences are updated. The change requests on session.model.preferences
+    // are also fired by the lifecycleManager session start and the contextManager, in which cases preferences should
+    // not be written to the cloud.
```

Comment:
Proper comment block here also - we should consider ourselves in a "modern era" where every significant function should be documented